heroku-api/src/main/java/com/heroku/api/Heroku.java

```diff
@@ -274,7 +270,7 @@ public String getHeaderName() {
 
         @Override
         public String getHeaderValue() {
-            return Integer.toString(version);
+            return "application/vnd.heroku+json; version=" + Integer.toString(version);
```

Comment:
While you're touching this, you might want to include the ability to set the version's variant.